components/MessageCard.vue

```diff
@@ -15,10 +15,57 @@
 <script>
 export default {
   props: ["message", "isThread"],
+
   methods: {
     showThread() {
       this.$emit('showThread', this.message.ts);
     },
-  }
+    replaceLinkWithATag(actualLink, displayLink){
+      var htmlTagLink = actualLink.link(actualLink);
```

Comment:
varは再宣言可能な変数宣言のため、letもしくはconstを検討してみてください。
```suggestion
      let htmlTagLink = actualLink.link(actualLink);
```

Reply:
修正しました。https://github.com/takuuuu517/nuxt-sample-app/pull/3/commits/badc495552ba363893f5639d9251673986affc1f